### DISCUSSION TITLE
Dynamische Mailempfänger yform

### DISCUSSION THREAD

**Discussion by studiosinmotion** [FriendsOfREDAXO/.github#12] [Q&A] [answered]
Hallo Zusammen,
ich hoffe hier kann mir jemand weiterhelfen, ich versuche über yform eine Mail an unsere Kunden zusenden. 

Die Empfängeradresse soll die, die im Formular eingetragene Adresse sein. 

Wie genau muss "tpl2email" aufgebaut sein, damit der Kunde eine Bestätigungsmail erhält? 

Viele Grüße

**Comment by tyrant88** [comment, marked as answer]
`action|tpl2email|emailtemplate|namedesemailformularfeldes`

bzw. in PHP:

`$yform->setActionField('tpl2email', array("emailtemplate", "namedesemailformularfeldes"));`

**Reply by eaCe** [reply]
@studiosinmotion wenn die Antwort für dich passt markiere diese doch bitte als beantwortet :) 

![grafik](https://user-images.githubusercontent.com/2708231/179959192-786de29a-c3fc-4ff5-a337-604e2ad796e7.png)
